go.mod

```diff
@@ -90,6 +90,7 @@ require (
 	github.com/yuin/goldmark-meta v1.1.0 // indirect
 	github.com/zclconf/go-cty v1.15.0 // indirect
 	go.abhg.dev/goldmark/frontmatter v0.2.0 // indirect
+	go.temporal.io/cloud-sdk v0.2.0 // indirect
```

Comment:
Surprised to see this under the `//indirect` section. Maybe run a `go mod tidy` and see if it causes updates?